x11-themes/numix-gtk-theme/numix-gtk-theme-9999.ebuild

```diff
@@ -1,33 +1,26 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 2004-2018 Sabayon
```

Comment:
This repository is for Gentoo so it needs to be QA compliant with the Gentoo copyright.